src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -50,9 +50,22 @@ const HeaderUserMenu = ({ user }) => {
               ''
             )}
             {!user.hasClientManager && !user.plan.buttonUrl ? (
-              <button onClick={() => toggleModal(true)} className="user-plan">
-                {user.plan.buttonText}
-              </button>
+              !user.isLastPlanRequested ? (
```

Comment:
This property is not yet in production `user.isLastPlanRequested` in GetUserData call. You are aware right? This change won`t break something?

Reply:
You are right, this pr should be merged when the Doppler version 22.5 is deployed in production, I add a label (no merge)